app/livechat/server/lib/Livechat.js

```diff
@@ -143,6 +143,12 @@ export const Livechat = {
 
 		if (newRoom) {
 			Messages.setRoomIdByToken(guest.token, room._id);
+			const message = {
+				t: 'livechat-started',
+				msg: '',
+				groupable: false,
+			};
+			sendMessage(guest, message, room);
```

Comment:
```suggestion
			sendMessage(guest, { t: 'livechat-started', msg: '', groupable: false }, room);
```

Also, the best place to implement this message is here:
https://github.com/RocketChat/Rocket.Chat/blob/9ff1e16147748d61100647928521b2cb42031cef/app/livechat/server/lib/Helper.js#L24